sapi/cli/php_http_parser.c

```diff
@@ -623,6 +624,8 @@ size_t php_http_parser_execute (php_http_parser *parser,
         } else if (parser->method == PHP_HTTP_MKCOL) {
           if (index == 1 && ch == 'O') {
             parser->method = PHP_HTTP_MOVE;
+          } else if (index == 3 && ch == 'A') {
+            parser->method = PHP_HTTP_MKCALENDAR;
```

Comment:
When control enters with `index=1` and `ch=K`, then [another else-if](https://github.com/php/php-src/blob/master/sapi/cli/php_http_parser.c#L613-L614) is executed, doing nothing.